easypermissions/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,3 @@
-<manifest package="com.google.example.easypermissions">
+<manifest package="pub.devrel.easypermissions">
```

Comment:
This seems to be a mistake: I couldn't get the project to find the `R` class with the old package name.